pkg/kubectl/cmd/util/printing.go

```diff
@@ -52,7 +52,7 @@ func AddOutputFlags(cmd *cobra.Command) {
 
 // AddNoHeadersFlags adds no-headers flags to a command.
 func AddNoHeadersFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("no-headers", false, "When using the default output, don't print headers.")
+	cmd.Flags().Bool("no-headers", false, "When using the default output, don't print headers (except for not default custom-column, it also works then).")
```

Comment:
How about `When using the default or custom-column output format, don't print headers.`